config/settings.yml

```diff
@@ -1,5 +1,5 @@
 sdr_api:
-  url: https://sdr-api-stage.stanford.edu
+  url: http://localhost:8080
```

Comment:
Changed so that won't provide false positives in other environments.

Reply:
Good call!